bin/dev

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env ruby
+exec './bin/rails', 'server', *ARGV
```

Comment:
or this one?

Reply:
It seems this is used in the context of https://github.com/rails/cssbundling-rails and https://github.com/rails/jsbundling-rails. We don't use either of the two but in case we need to make the switch, this file is here. I don't believe this breaks any workflows so no harm in keeping it.